### PR TITLE
Feature/orca 482 Update documentation for specifying storage class on ingest

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Update documentation for specifying storage class on ingest

Addresses [ORCA-482: Update documentation for specifying storage class on ingest](https://bugs.earthdata.nasa.gov/browse/ORCA-482)

## Changes

* Added storage-classes documentation and linked to it in relevant locations.

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: website/docs/operator/storage-classes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### GLACIER/Glacier Flexible Retrieval
+The default storage class. Files stored in the `GLACIER` class can be recovered in as quickly as 5 minutes using the `Expedited` [option](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html) as needed. Cheeper options include the orca default `Standard` at 5 hours, and the Bulk at up to 12 hours. For pricing details, see 'Requests & data retrievals' on the [Amazon S3 pricing](https://aws.amazon.com/s3/pricing/) page.
+
+### DEEP_ARCHIVE
+Files stored in `DEEP_ARCHIVE` [cost roughly 1/3rd](https://aws.amazon.com/s3/pricing/) to keep stored in S3 compared to `GLACIER`.
+As a trade-off, [retrieval takes 3-4 times as long](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html), and cannot be restored with the `Expedited` recovery type.
+It is also moderately more expensive to store files as `DEEP_ARCHIVE`, but [the cost is still minimal](https://aws.amazon.com/s3/pricing/).
```
Comment (bhazuka) [inline website/docs/operator/storage-classes.md:18]: `moderately more expensive to store files as `DEEP_ARCHIVE`` shouldn't this be less? In the sentence above you state the cost is ~1/3 less than Glacier. I wonder if you meant restoration here? Although I think it costs less possibly (at least for bulk) not sure for standard.

**Review by bhazuka — CHANGES_REQUESTED** [review]
Overall looks good. Found on minor item.

File: website/docs/operator/storage-classes.md (hunk 1 above)
Comment (AndrewDorn) [reply website/docs/operator/storage-classes.md]: Ongoing storage costs less, but putting the file in the bucket costs more as DEEP_ARCHIVE. I've attempted to clarify the language.

File: website/docs/operator/storage-classes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description: Provides a brief overview on S3 storage classes available in Orca.
+---
+
+Orca can accept multiple [S3 storage classes](https://aws.amazon.com/s3/storage-classes/) on ingest.
+These storage classes affect storage pricing, as well as available [retrieval options](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html).
+
+See [Deployment With Cumulus](../developer/deployment-guide/deployment-with-cumulus.md) for details on choosing a default storage class and [Collection Configuration](collection-configuration.md) for setting the per-collection override value.
+
+### GLACIER/Glacier Flexible Retrieval
+The default storage class. Files stored in the `GLACIER` class can be recovered in as quickly as 5 minutes using the `Expedited` [option](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html) as needed. Cheeper options include the orca default `Standard` at 5 hours, and the Bulk at up to 12 hours. For pricing details, see 'Requests & data retrievals' on the [Amazon S3 pricing](https://aws.amazon.com/s3/pricing/) page.
```
Comment (rizbihassan) [inline website/docs/operator/storage-classes.md:13]: fix Cheeper spelling

File: website/docs/operator/storage-classes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### DEEP_ARCHIVE
+Files stored in `DEEP_ARCHIVE` [cost roughly 1/3rd](https://aws.amazon.com/s3/pricing/) to keep stored in S3 compared to `GLACIER`.
+As a trade-off, [retrieval takes 3-4 times as long](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html), and cannot be restored with the `Expedited` recovery type.
+The initial cost of placing or updating a file in `DEEP_ARCHIVE` is moderately more expensive compared to `GLACIER`, but [is still minimal unless you are uploading thousands of files](https://aws.amazon.com/s3/pricing/).
```
Comment (rizbihassan) [inline website/docs/operator/storage-classes.md:18]: I think deep archive is the cheapest but user has to wait upto 12 hours before they can access the files.

**Review by rizbihassan — CHANGES_REQUESTED** [review]

File: website/docs/operator/storage-classes.md (hunk 2 above)
Comment (AndrewDorn) [reply website/docs/operator/storage-classes.md]: Fixed

File: website/docs/operator/storage-classes.md (hunk 3 above)
Comment (AndrewDorn) [reply website/docs/operator/storage-classes.md]: Removed

**Review by rizbihassan — APPROVED** [review]

**Review by bhazuka — APPROVED** [review]
Looks good to me

**Pull request merged**